tests/e2e/utils/workspace/ApiUrlResolver.ts

```diff
@@ -0,0 +1,50 @@
+/*********************************************************************
+ * Copyright (c) 2019 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+import { inject, injectable } from 'inversify';
+import { CLASSES } from '../../inversify.types';
+import { Logger } from '../Logger';
+import { CheApiRequestHandler } from '../requestHandlers/CheApiRequestHandler';
+
+@injectable()
+export class ApiUrlResolver {
+    static USER_NAMESPACE: string = '';
+
+    private dashboardApiUrl: string = 'dashboard/api/namespace';
+    private kubernetesApiUrl: string = 'api/kubernetes/namespace';
```

Comment:
`kubernetesApiUrl` is actually the constant, so it makes sense to capitalize the name and declare as `static readonly`, following up code convention.

Reply:
fixed

Reply:
thanks